vignettes/sidebars.Rmd

```diff
@@ -58,215 +87,398 @@ iframe, .bslib-page-fill {
   border: var(--bs-border-width) solid var(--bs-border-color);
   margin-bottom: 4rem;
   box-shadow:
-    rgba(0, 0, 0, 0.25) 0px 54px 55px, 
-    rgba(0, 0, 0, 0.12) 0px -12px 30px, 
-    rgba(0, 0, 0, 0.12) 0px 4px 6px, 
-    rgba(0, 0, 0, 0.17) 0px 12px 13px, 
+    rgba(0, 0, 0, 0.25) 0px 54px 55px,
+    rgba(0, 0, 0, 0.12) 0px -12px 30px,
+    rgba(0, 0, 0, 0.12) 0px 4px 6px,
+    rgba(0, 0, 0, 0.17) 0px 12px 13px,
     rgba(0, 0, 0, 0.09) 0px -3px 5px;
 }
+
+iframe {
+  margin-bottom: calc(55px + 2rem);
+}
+
+iframe.resizable {
+  resize: vertical;
+  min-height: 300px;
+}
+
+.card-header.bg-info code {
+  background: none;
+  color: hsl(209deg, 85%, 25%);
+}
 ```
 
-### Sidebar layouts
+::: lead
+Sidebar layouts in web interfaces allow your users to easily access filters, settings and other inputs alongside the interactive features they control.
+In this article, you'll learn how to create sidebar layouts with bslib.
+:::
+
+## Introduction
+
+To create a sidebar layout, provide a `sidebar()` to the `sidebar` argument of one of the following functions.
+
+:::: {.row}
+::: {.col-sm-12 .col-lg-6}
+### Shared Sidebar {#intro-shared-sidebar}
+
+When `sidebar` is an argument to a function containing multiple pages or panels, it creates a **[shared sidebar](#shared-sidebar)**.
```

Comment:
Since you use tabs below, better to be consistent

```suggestion
When `sidebar` is an argument to a function containing multiple pages or tabs, it creates a **[shared sidebar](#shared-sidebar)**.
```